Tensile/Common.py

```diff
@@ -1157,6 +1198,11 @@ def getArchitectureName(gfxName):
     {"MaxVgprNumber":             [256]},
     {"StoreRemapVectorWidth":     [ 0 ] },
     {"SourceSwap":                [ False ] },
+    {"StorePriorityOpt":          [ False ] },
+    {"NumElementsPerBatchStore":  [ 0 ] },
+    {"StoreSyncOpt":              [ 0 ] },
+    {"GroupLoadStore":            [ False ] },
+    {"InterleaveStoreVmcnt":      [ True ] },
     ]
```

Comment:
are these parameters must be enabled to interleave store with compute from other wave to get good performant kernel?? if yes, can you define single parameter that generates kernel 'interleave store(1st wave) with compute (2nd wave) and convert all these parameters into variables and assign optimum values. 

is there optimum number for s_sleep ?? is that controlled by parameter?

Reply:
I am testing different sizes to see performance impact with these parameters.
here are parameters I used to tune for HPL sizes
- StorePriorityOpt: [1]
- NumElementsPerBatchStore: [2,4]
- StoreSyncOpt: [0,1,2,4,6]
- GroupLoadStore: [1]
- InterleaveStoreVmcnt: [0]